### PR TITLE
Add step to clone training repository missing from current instructions

### PR DESCRIPTION
**Pull request by jdlangs**
This resolved the issue that was reported on the mailing list here: https://groups.google.com/forum/#!topic/swri-ros-pkg-dev/FtoQ3u7-HDI

### CODE REVIEW

File: gh_pages/_source/setup/PC-Setup---ROS-Kinetic.md [hunk 1]
```diff
@@ -23,8 +23,12 @@ The Virtual Box is limited both in hardware capability(due to VM limitations) an
 On most new systems, Virtual Box and VMs work out of the box.  The following is a list of issues others have encountered and solutions:
   * Virtualization must be enabled - Older systems do not have virtualization enabled (by default).  Virtualization must be enabled in the BIOS.  See <http://www.sysprobs.com/disable-enable-virtualization-technology-bios> for more information.
 
-## PC Configuration (**NOT Recommended**)
-An installation [shell script](https://github.com/AustinDeric/ros-industrial-env) is provided to run in Ubuntu Linux 16.04 (Xenial Xerus) LTS.  This script loads all the programs needed and configures the environment.
+## Direct Linux PC Configuration (**NOT Recommended**)
+An installation [shell script](https://github.com/AustinDeric/ros-industrial-env) is provided to run in Ubuntu Linux 16.04 (Xenial Xerus) LTS.  This script installs ROS and any other packages needed for the environment used for this training.
```
Comment (jrgnicho) [inline gh_pages/_source/setup/PC-Setup---ROS-Kinetic.md]: Not sure how I feel about pointing to a script in a personal repo.  Could we copy this file into this repository? with @AustinDeric consent of course.

Comment (jdlangs) [reply gh_pages/_source/setup/PC-Setup---ROS-Kinetic.md]: :+1: to that

Comment (jrgnicho) [reply gh_pages/_source/setup/PC-Setup---ROS-Kinetic.md]: @jdlangs ping

Comment (jrgnicho) [reply gh_pages/_source/setup/PC-Setup---ROS-Kinetic.md]: @jdlangs could you copy that script into this repo?

**Comment by jdlangs** [comment]
Have to open a new PR for this against `kinetic` now that `kinetic-devel` is gone.

**Pull request closed without merging**